src/components/elements/rebranding-header.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { TString } from 'types/generics'
+import { Localize } from 'components/localization'
+import { Flex } from 'components/containers'
+
+type RebrandingHeaderProps = {
```

Comment:
rename this filename to common-header-section

Reply:
So as the  component to CommonHeaderSection

Reply:
Ok 👍